### ISSUE TITLE
Persistent issues when running with clerk

### ISSUE THREAD

**Issue by abondrn** [zmedelis/bosquet#43]
I am currently trying to get bosquet set up.

When I have the following line in `deps.edn`:
`io.github.zmedelis/bosquet {:mvn/version "v2023.10.11"}`

I get this upon startup (same as #35, thought I would have expected this to be fixed in this version):
```
Execution error (NoSuchFieldError) at clojure.tools.analyzer.jvm.utils__init/load (REPL:259).
__thunk__0__

Full report at:
/var/folders/df/0xb_85b51dx2p538kv3mf1pw0000gn/T/clojure-2323675351862269305.edn
```

When I have `v2023.10.31`, clerk is able to start up fine, but the following namespace:
```
(ns llm
  (:require [nextjournal.clerk :as clerk]
            [bosquet.wkk :as wkk]
            [bosquet.llm.generator :as g]
            [bosquet.system :as system]
            ))
```

Fails with
```
Unhandled java.lang.Exception
namespace 'bosquet.system' not found
```

Note that this is only caused by the last two imports; importing `bosquet.wkk` is fine. This persists when this library is installed as `io.github.zmedelis/bosquet {:git/sha "00a1534f2dd9d274ae55f9f771f016522017e291"}`.

This is the command I am using to run clerk (ignore the escapes): `clj -M -e \"(require '[nextjournal.clerk :as clerk]) (clerk/serve! {:browser true :watch-paths [\\\"notebooks\\\" \\\"src/clj\\\" \\\"deps.edn\\\"]})\"`
I also have a `config.edn` and `deps.edn` and no other clojure configuration.

**Comment by sheluchin** [comment]
Can confirm, I saw the same behaviour.

**Comment by abondrn** [comment]
Fixed for me as of `{:git/sha "b62968ad76fa14f7faf0ffdf3cb08ba9f6d152cc"}`, feel free to close after the next stable release

**Comment by zmedelis** [comment]
Thanks for reporting!

It is now part of `{:mvn/version "v2023.11.21"}` (why did I add the 'v' in the version name 🤔)
